meetuprepo/src/main/java/com/gdgistanbul/repo/MeetupRepo.kt

```diff
@@ -1,13 +1,18 @@
 package com.gdgistanbul.repo
 
 import com.gdgistanbul.api.MeetupApi
+import com.gdgistanbul.api.MeetupSecureApi
 import com.gdgistanbul.model.Event
+import com.gdgistanbul.model.Login
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
 
-class MeetupRepo(val meetupApi: MeetupApi) {
+class MeetupRepo(val meetupApi: MeetupApi, val meetupSecureApi: MeetupSecureApi) {
     suspend fun getEvents(): List<Event> = withContext(Dispatchers.IO) {
         meetupApi.getEvents()
     }
 
+    suspend fun getAccessToken(code: String): Login = withContext(Dispatchers.IO) {
+        meetupSecureApi.getAccessToken(code = code)
```

Comment:
`meetupSecureApi.getAccessToken` takes 4 mandatory parameters, just `code` is explicit.